lms/tasks/email_digests.py

```diff
@@ -11,7 +11,7 @@
 LOG = logging.getLogger(__name__)
 
 
-@app.task
+@app.task(max_retries=2, autoretry_for=(Exception,))
```

Comment:
Todo: add exponential backoff and jitter. We may also want to use manually retrying so that we don't re-process user IDs that we already successfully processed